### PR TITLE
[Admin] Define DemoTests

### PR DESCRIPTION
**Pull request by TessaPeruzzi**
DemoTests for #55

### CODE REVIEW

**Review by brendanfraser597 — DISMISSED** [review]

File: docs/uml-class-diagrams/admin/test/DemoTests/DemoTests.puml [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 package edu.oakland.test.admin {
     class DemoTests {
-
+        + void rfidNumberIsNotNull()
+        + void storageInputLocIsNotNull()
+        + void storageOutputLocIsNotNull()
```
Comment (andrewdimmer) [inline docs/uml-class-diagrams/admin/test/DemoTests/DemoTests.puml]: Just occurred to me that rfid is an int... and thus it can't be null. Then it occurred to me than an empty string is another edge case we need to handle. What do you think of this?
```suggestion
        + void storageInputLocIsNotNull()
        + void storageOutputLocIsNotNull()
        + void storageInputLocIsNotEmpty()
        + void storageOutputLocIsNotEmpty()
```

**Review by andrewdimmer — CHANGES_REQUESTED** [review]
One quick change to handle empty strings, then I think you're set to merge!

**Review by andrewdimmer — APPROVED** [review]
Looks good!

**Review by brendanfraser597 — APPROVED** [review]
